cli/viam/main.go

```diff
@@ -103,7 +106,7 @@ func main() {
 				Subcommands: []*cli.Command{
 					{
 						Name:  "print-access-token",
-						Usage: "print-access-token - print an access token for your current credentials",
+						Usage: "print an access token for your current credentials",
```

Comment:
Q: Can this ever be more than one token? If not:

```suggestion
						Usage: "print the access token associated with your current credentials",
```

Reply:
> Can this ever be more than one token?

From what I can tell, no. Updated to the above.